exporter/tile38.go

```diff
@@ -1,20 +1,26 @@
 package exporter
 
 import (
+	"strings"
+
 	"github.com/gomodule/redigo/redis"
 	"github.com/prometheus/client_golang/prometheus"
 	log "github.com/sirupsen/logrus"
 )
 
 func (e *Exporter) extractTile38Metrics(ch chan<- prometheus.Metric, c redis.Conn) {
-	info, err := redis.Strings(doRedisCmd(c, "SERVER"))
+	info, err := redis.Strings(doRedisCmd(c, "SERVER EXT"))
 	if err != nil {
 		log.Errorf("extractTile38Metrics() err: %s", err)
 		return
 	}
 
 	for i := 0; i < len(info); i += 2 {
-		fieldKey := "tile38_" + info[i]
+		fieldKey := info[i]
+		if !strings.HasPrefix(fieldKey, "tile38_") {
+			fieldKey = "tile38_" + fieldKey
```

Comment:
Thanks for jumping in @houstonheat  - I agree, extending the tests a little to cover more metrics is a good idea.

Reply:
@houstonheat thank you for your comment, I think having a metric with double tile38_tile38_ is not possible because I check the prefix with `strings.HasPrefix(fieldKey, "tile38_")`  (and tile38 doesn't expose such metrics either) so I think this case would not happen but I added a regex metric pattern to verify tile38 metrics. If you feel it is still incomplete, tag me to write a more complicated pattern.

Reply:
Oh, sorry, I think I misread the original comment by @houstonheat  - I think the code is fine like this.

Reply:
But can you add a test that one of the metrics that originally does not start with `tile38_` is getting picked up due to this `if` statement? 
(related to expanding the tests, see comment below)

Reply:
@oliver006 thank you for your review, I added few more metrics for this like `tile38_cpus_total ` which was exported as `sys_cpus`, and new metrics like `tile38_go_goroutines_total` and some metric that is changed to be compatible with the previous version like `tile38_avg_item_size_bytes`.
(I run tests locally this time :))) )